middle_end/flambda2/tests/mlexamples/array_specialisation_examples.ml

```diff
@@ -0,0 +1,33 @@
+(* Should be simplified to a Values array load, but isn't because of the missing
+   representation for Values arrays in Flambda kinds *)
```

Comment:
This comment is out of date now, I think?

Reply:
Yes, I'll update the comments in this file (and check that it does get simplified to a Values load).